### ISSUE TITLE
Can't open specific image with no errors

### ISSUE THREAD

**Issue by rogday** [ArturKovacs/emulsion#35]
Can't open this file, but default windows "Photos" app works fine. Title of emulsion just says "[ none ]". There is no crash dump or anything.
OS: Windows 10
Emulsion v1.9
https://dropmefiles.com/YOVjO

**Comment by tuncer** [comment]
@ArturKovacs, maybe we need a small `try_open_with_image_rs` program that could help debug issues with image-rs itself. Perhaps they already have something like that in their examples/tests. Thoughts?

**Comment by ArturKovacs** [comment]
This is indeed a bug in the `image-rs` create. I posted the issue at: https://github.com/image-rs/jpeg-decoder/issues/151

-------

> a small `try_open_with_image_rs` program

Well here's one in case anyone needs it but I don't think there's gonna be too many of these types of bug reports. (Narrator's voice: but little did he know...)

```rust
use image;

fn main() {
    let args: Vec<_> = std::env::args().collect();
    if let Some(path) = args.get(1) {
        image::open(path).unwrap();
        println!("Image loaded without errors.");
    } else {
        println!("Didn't specify file path. Correct usage:\n{} path/to/image/file", args.get(0).unwrap());
    }
}
```

**Comment by ArturKovacs** [comment]
Fixed by 751e07bd17a380b1835d42001c28bae199ae9f7c
